core/shared/platform/common/posix/posix_time.c

```diff
@@ -15,3 +53,28 @@ os_time_get_boot_microsecond()
 
     return ((uint64)ts.tv_sec) * 1000 * 1000 + ((uint64)ts.tv_nsec) / 1000;
 }
+
+uint64
+os_clock_res_get(bh_clock_id_t clock_id, uint64 *resolution)
+{
+    bh_clock_id_t nclock_id;
```

Comment:
```suggestion
    clockid_t nclock_id;
```
because `convert_clockid` returns posix's `clockid_t` value